### ISSUE TITLE
Content jumps when navbar sticks

### ISSUE THREAD

**Issue by ejerskov** [jd1378/vue-sticky-element#29]
Im using the sticky element with my navbar, like this:

```
  <StickyElement
    visible-on-direction="disabled"
    stick-mode="element-start"
  >
    <Navbar />
</StickyElement>
```

When the Navbar sticks, the content below will jump because of the missing offset. How can I controls this?
![scroll](https://github.com/user-attachments/assets/15ae8192-051c-48d0-8e35-8a8760824849)

I have tried to add some margin to the content when @stuck is triggered, but when scrolling the event is triggerede multiple times which makes content jump a lot

**Comment by jd1378** [comment]
Hi
Can you make a reproduction using [codesandbox](https://codesandbox.io/) or [stackblitz](https://stackblitz.com/)?

**Comment by ejerskov** [comment]
I got it to work by added and div underneath with the same height as the navbar. Its hidden until the navbar is stuck, so it fills up the void left after position: fixed.

I just think that the plugin should do this by itself as this must be a common way to use it

**Comment by jd1378** [comment]
The plugin detects the height of the element and uses it on the wrapper, so it being stuck or not should not move the content, thats why I find it odd

I'm trying to reproduce at the moment to see why its not working as expected

**Comment by jd1378** [comment]
this is my test and it seems to be working fine:
https://codesandbox.io/p/sandbox/vue-sticky-element-vue3-forked-7wny8h

**Comment by ejerskov** [comment]
Oh, it seems to be caused by a <div class="flex"> wrapped around it, so the divs height is ignored.
https://anfieldroad.dk/

Do you have a suggestion to how to fix this?

**Comment by jd1378** [comment]
can you recreate it in either of the websites I mentioned ? I want to play around with it so I can understand it better,
this way its a bit hard to understand for me

**Comment by jd1378** [comment]
hmm, for some reason, even with height:64px, the height of it is becoming 0. investigating why ...

**Comment by jd1378** [comment]
okay I think I figured it out, I'll try to release a fix soon

**Comment by ejerskov** [comment]
Awesome, thanks!

**Comment by jd1378** [comment]
it has been released in 2.0.3
let me know if it doesn't work properly :)

**Comment by ejerskov** [comment]
Its working, great stuff! Thank you for the quick response.
